testdata/p4_16_samples_outputs/structured-annotation.p4-stderr

```diff
@@ -0,0 +1 @@
+warning: Program does not contain a `main' module
```

Comment:
There is no test program that contains a generated P4Info file.  Is it planned that one will be added later after appropriate changes are made in the PI project on which p4c depends?  Or should that be possible to create now?

Reply:
Answering my own comment: I am pretty sure that this PR, or something that similarly updates the .proto files defining the allowed contents of the P4Info files, would need to be merged in first, before it would be possible to write an automated test for p4c that checks that P4Info files are written by p4c with the contents of structured annotations: https://github.com/p4lang/p4runtime/pull/265

So any such tests for p4c will have to wait for that merging to be done first.

Reply:
@jafingerhut the plan is to separate IR changes (which Mihai is taking care of) from the P4Info generation changes (which I will take care of).

Reply:
Sounds good.  Resolving this conversation on Github.